schema.graphql

```diff
@@ -134,6 +134,7 @@ enum Interaction {
   UNLIST
   LOCK
   CHANGEISSUER
+  PAYTIPS
```

Comment:
Please use the one from kodadot/snek

Reply:
hmmmmm...
what if we'll create feature in futur to tip artist? this will be show as `PAY_ROYALTY`

to me royalty are cut of price not price + royalty

Reply:
Then you can differentiate if it was part of some batch or single ☺️